### ISSUE TITLE
✅ 🐞 Floating button username for Messenger is not working on Mobile

### ISSUE THREAD

**Issue by reedock** [elementor/elementor#28103]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

By default the link used for Messenger is https://www.facebook.com/messages/t/ and there is a need to add a "username"

It should be https://m.me/ "name of page / username" instead so it work for both mobile and regular browsers.

See screenshot, it doesn't work on mobile because user is not found:
https://ibb.co/cQSrd73

If I use a simple icon widget with the following URL below, it open properly a new conversation with any devices:
https://m.me/ "name of my Facebook page"

### Steps to reproduce

1. Create floating button
2. Add name of your Facebook page
3. Using mobile device, it doesn't open a conversation with a Facebook page and give this error:
https://ibb.co/cQSrd73

### Expected behavior

**The link of this widget is by default**:
https://www.facebook.com/messages/t/ _add name of your page / username_
**It should be**:
https://m.me/ _add name of your page / username_

If I use a simple icon widget with the following URL below, it open properly a new conversation:
https://m.me/ "name of page or username"

**REQUEST FEATURE**:
Add the possibility to customize our own URL & Icons for the Resource links would be better to allow multiple scenarios.

**FROM OUR FRIEND, CHATGPT**:

What's the difference between the 2 URLs?

https://www.facebook.com/messages/t/
&
https://m.me/

Both URLs are used to access Facebook Messenger, but there are subtle differences in their usage and appearance:

**https://www.facebook.com/messages/t/**

This URL directs you to the Facebook Messenger interface within the Facebook website.
When you visit this link, you need to be logged into your Facebook account to view and send messages.
It is generally used when accessing Messenger from a desktop or a laptop.

**https://m.me/**

This URL provides a direct link to a chat with the specified Facebook page on Messenger.
It can be used on both desktop and mobile devices, but it is particularly useful for mobile users as it can open the Messenger app directly if it is installed.
It is a simpler, more universal link that doesn't require navigating through the main Facebook website.
In summary, while both links will take you to Facebook Messenger to chat with the specified page, the first is a direct link to the messages interface on Facebook's website, and the second is a more streamlined link that is optimized for both mobile and desktop use, often opening the Messenger app on mobile devices.

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MariaDB Server v10.6.18
	PHP Version: 8.0.30
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 2000
	PHP Max Post Size: 1024M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.5.5
	WP Multisite: No
	Max Upload Size: 1 GB
	Memory limit: 512M
	Max Memory limit: 1024M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: Asia/Manila
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.1.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36

== Active Plugins ==
	Code Snippets
		Version: 3.6.5.1
		Author: Code Snippets Pro

	Elementor
		Version: 3.23.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.23.0
		Author: Elementor.com

	LiteSpeed Cache
		Version: 6.2.0.1
		Author: LiteSpeed Technologies

	UpdraftPlus - Backup/Restore
		Version: 1.24.4
		Author: UpdraftPlus.Com, DavidAnderson

	Wordfence Security
		Version: 7.11.6
		Author: Wordfence

	Yoast SEO
		Version: 23.0
		Author: Team Yoast


== Must-Use Plugins ==
	WP.com Site Helper
		Version: 
		Author: 


== Features ==
	Custom Fonts: 2
	Custom Icons: 0

== Integrations ==
	
	recaptcha_v3: Active


== Elementor Experiments ==
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active
	Flexbox Container: Active
	Grid Container: Active by default
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Control Loading: Active
	Optimized Markup: Inactive by default
	Plugin Onboarding: Active by default
	Default to New Theme Builder: Active by default
	Header & Footer: Active by default
	Elementor Home Screen: Active by default
	Editor Top Bar: Inactive by default
	Build with AI: Active by default
	Landing Pages: Active by default
	Nested Elements: Active
	Lazy Load Background Images: Active by default
	Element Caching: Active
	Link In Bio: Inactive by default
	Floating Buttons: Active
	Elementor Editor Events: Inactive by default
	Display Conditions: Active by default
	Form Submissions: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Active by default
	Search: Inactive by default


== Log ==
	
Log: showing 20 of 422024-06-26 18:14:21 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.1',
  'to' => '3.22.3',
)]
2024-06-26 18:14:21 [info] elementor::elementor_updater Started 
2024-06-26 18:14:21 [info] Elementor/Upgrades - _on_each_version Start  
2024-06-26 18:14:21 [info] Elementor/Upgrades - _on_each_version Finished 
2024-06-26 18:14:21 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.1',
  'to' => '3.22.3',
)]
2024-06-26 18:14:23 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.1',
  'to' => '3.22.3',
)]
2024-07-15 23:10:55 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.22.1',
  'to' => '3.23.0',
)]
2024-07-15 23:10:55 [info] elementor-pro::elementor_pro_updater Started 
2024-07-15 23:10:55 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-07-15 23:10:55 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-07-15 23:10:55 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.22.1',
  'to' => '3.23.0',
)]
2024-07-15 23:11:37 [info] elementor::elementor_updater Started 
2024-07-15 23:11:37 [info] Elementor/Upgrades - _on_each_version Start  
2024-07-15 23:11:37 [info] Elementor/Upgrades - _on_each_version Finished 
2024-07-15 23:11:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.22.3',
  'to' => '3.23.0',
)]
2024-07-16 12:45:38 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.23.0',
  'to' => '3.23.1',
)]
2024-07-16 12:45:38 [info] elementor::elementor_updater Started 
2024-07-16 12:45:38 [info] Elementor/Upgrades - _on_each_version Start  
2024-07-16 12:45:38 [info] Elementor/Upgrades - _on_each_version Finished 
2024-07-16 12:45:38 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.23.0',
  'to' => '3.23.1',
)]

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @reedock! 

Thank you for submitting your issue! 🙏 

We are aware of this issue and our team is already working on a fix. We can’t say exactly when it'll be sorted out, but we kindly ask for your patience. Thanks for understanding! 😉

Best regards

**Comment by nicholaszein** [comment]
@reedock, please, keep in mind that GitHub Issues are reserved only for Bug Reports. Do not use issues to request features or support. For feature requests go to https://elemn.to/gh-feature-requests. For Personal Support login to your https://my.elementor.com Account and request support from there. Thanks

**Comment by reedock** [comment]
@nicholaszein Thank you for your message really appreciate knowing that you are working on a fix. 

My bad. It was more of a hint to create a more future proof solution. 

If you update this to https://m.me/, which works on all devices today, but Meta changes the URL **again** tomorrow, we’ll have to wait for someone to notice it and then wait for an update from you guys. That could take days/weeks and that's millions of websites that will be affected..

Will be adding the feature request. :)

**Comment by nicholaszein** [comment]
Hey @reedock!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.23.2! 🥳 

All Facebook Messenger links now use the https://m.me/ link! 💪 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
